compaction/task_manager_module.hh

```diff
@@ -62,9 +62,11 @@ public:
             std::string table,
             std::string entity,
             tasks::task_id parent_id,
-            flush_mode fm = flush_mode::compacted_tables) noexcept
+            flush_mode fm = flush_mode::compacted_tables,
+            bool force_purge_tombstones = false) noexcept
         : compaction_task_impl(module, id, sequence_number, std::move(scope), std::move(keyspace), std::move(table), std::move(entity), parent_id)
-        , _flush_mode(fm)
+        , _flush_mode(force_purge_tombstones ? flush_mode::all_tables : fm)
```

Comment:
When called from `global_major_compaction_task_impl::run` we already flush all memtables so currently we pass `flush_mode::skip`.  I'm not sure that the `force_purge_tombstones` should change this convention.  If provided for `/storage_service/compact`, it should be ok to flush all memtables once for all keyspaces.

Reply:
I have retained the old behaviour. The new version ensures that even with aggressive_purge_tombstones enabled, the tables are flushed only once when called from global_major_compaction.